.evergreen/evergreen.yml.in

```diff
@@ -1734,6 +1789,11 @@ buildvariants:
   - name: draft_publish_release
     display_name: "Draft/Publish Release"
     run_on: ubuntu2004-small
+    depends_on:
+      <% for (const { packages } of RELEASE_PACKAGE_MATRIX) {
+       for (const { name: packageVariant } of packages) { %>
+      - name: verify_artifact_<% out(packageVariant.replace(/-/g, '_')) %>
+      <% } } %>
```

Comment:
We generally don't set `depends_on` in the build variants section, there's an existing one for the `release_draft` task specifically that shows what the task dependencies look like:

https://github.com/mongodb-js/mongosh/blob/590e5725265723d752048cff37b29209ac4cfd15/.evergreen/evergreen.yml.in#L1267-L1279

So I think it should be enough to tag the `verify_artifact_...` tasks with the `smoke-test` tag